NuRadioReco/framework/sim_channel.py

```diff
@@ -14,11 +14,11 @@ class SimChannel(NuRadioReco.framework.channel.Channel):
     """
     Object to store simulated channels.
 
-    This class is the same as the regular channel trace but has apart from the channel id also
+    This class is the same as the regular channel trace but has apart from the channel id (and possible group id) also
     a shower and ray tracing solution id
     """
 
-    def __init__(self, channel_id, shower_id, ray_tracing_id):
+    def __init__(self, channel_id, shower_id, ray_tracing_id, channel_group_id):
```

Comment:
Please set the default argument to None (just copy what is done in Channel on the LOFAR branch). An remember to update the docstring accordingly (you can also copy that from the LOFAR branch docstring if you like)

Reply:
I set the default argument and copied the docstring from the LOFAR branch, thanks.